### PR TITLE
Quick-Create-Antagonist tweaks

### PR DESCRIPTION
**Pull request by bloons3**
## About The Pull Request
Implements https://github.com/BeeStation/BeeStation-Hornet/pull/3950

## Why It's Good For The Game

Gives admins control of how many antags they spawn, instead of assuming a maximum of (generally) 3.
image

## Changelog
:cl:
admin: Create Antag now allows for a maximum amount of antags to be selected for some options.
/:cl:

### CODE REVIEW

**Comment by OceanFish1** [comment]
tgstation 13 but every single player is valid

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -55,33 +55,37 @@
 			return
 		switch(href_list["makeAntag"])
 			if("traitors")
-				if(src.makeTraitors())
-					message_admins("[key_name_admin(usr)] created traitors.")
-					log_admin("[key_name(usr)] created traitors.")
+				var/maxCount = input("Set number of Traitors","Set Traitor Count (max)",1) as num|null
```
Comment (LemonInTheDark) [inline code/modules/admin/topic.dm:58]: why set a default of 1? why not set it to 0 and have the make(whatever) procs handle it?

Comment (bloons3) [reply code/modules/admin/topic.dm:58]: Cancelling should correctly set null.
1 is the default since it is assumed you want at least 1, or to cancel.

Comment (LemonInTheDark) [reply code/modules/admin/topic.dm:58]: aight

**Pull request merged**